api/v1/models/monitor_status.go

```diff
@@ -0,0 +1,45 @@
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	strfmt "github.com/go-openapi/strfmt"
+
+	"github.com/go-openapi/errors"
+)
+
+// MonitorStatus Status of the node monitor
+// swagger:model MonitorStatus
+type MonitorStatus struct {
+
+	// The number of CPUs to listen on for events.
```

Comment:
Fix the godoc comment, by starting with the name of the field:

    // Cpus is the number ...